src/components/TodoCreationForm.js

```diff
@@ -1,27 +1,52 @@
 import React from "react";
+import DatePicker from "react-datepicker";
+import Button from "./Library/Button";
 
 export default class TodoCreationForm extends React.Component {
   state = {
-    taskName: ""
+    todoName: "",
+    dueDate: new Date()
   }
 
+  handleDueDateChange = date => {
+    this.setState({
+      dueDate: date
+    });
+  };
+
   render() {
     return (
       <form
+        className="todo-creation-form"
         onSubmit={e => {
           e.preventDefault();
-          if (this.state.taskName === "") return;
-          this.props.onCreate(this.state.taskName)
-          this.setState({taskName: ""})
+          if (this.state.todoName === "") return;
+          this.setState({todoName: ""})
         }}
       >
         <input
           className="input is-primary is-small"
           autoFocus
-          placeholder="Add a task (hit Enter to validate)"
-          value={this.state.taskName}
-          onChange={e => this.setState({taskName: e.target.value})}
+          placeholder="Add a todo (hit Enter to validate)"
+          value={this.state.todoName}
+          onChange={e => this.setState({todoName: e.target.value})}
+        />
+        <DatePicker
+          selected={this.state.dueDate}
```

Comment:
I suggest we set the initial due date to `null` in the initial state. It is weird to auto select a due date ourselves because we have no idea what the user wants.
We can add a placeholder text like `"Enter a date"`, which will be useful anyway. 
Check in the ReactDatepicker library how to add a placeholder :)